packages/reactivue/src/component.ts

```diff
@@ -27,12 +40,17 @@ export const createNewInstanceWithId = (id: number, props: any, data: Ref<any> =
     _id: id,
     props,
     data,
+    isActive: false,
```

Comment:
What's this for?

Reply:
React Strict mode running state functions twice on dev. Including our createState.

createState creates a new instance. That means it create 2 instance for single component. But only the second state object will be used by us. To determine which instances are stale which ones are actually living with our components, I'm checking are they mounted or not. When our component gets mounted I set this value to true. And I know, every instances who has isActive === false are stale. They are initialized but never mounted. If we dont do check, their effects, like (useMouse for vueuse) will get updated while instances itself is not bound to any component. That causes memory leak in dev. 

`Object.keys(_vueState).forEach(id => !_vueState[+id].isActive && unmount(+id))`

In here, before creating new instance id, I remove the stale ones first for incremental instance ids.

Reply:
Thanks for the explanation, make sense to me now